profile.html

```diff
@@ -31,39 +31,63 @@
     <main class="bg-primary">
 
         <div class="container">
-            <div class="row">
+            <div class="row pt-5 d-flex justify-content-center">
+
+                <div class="col-4">
+                    <img src="images/profile-picture.jpg" class="rounded-circle logo">
+                    <h2>Christiane Dragvik</h2>
+                </div>
     
-                <div class="border bg-secondary col-4">
+                <div class="border bg-secondary col-3 m-2">
+                    <h3>Profile information</h3>
                     <div class="d-flex">
                         <p>Asker, Norway</p>
                     </div>
                     <div class="d-flex">
                         <p>Student at Noroff</p>
                     </div>
                     <div class="d-flex">
-                        <p>email@email.com</p>
+                        <p>cndragvik@gmail.com</p>
                     </div>
                     <div class="d-flex">
-                        <p>+00 0000 0000</p>
+                        <p>+47 968 29 487</p>
                     </div>
                 </div>
 
-                <div class="col-4">
-                    <img src="images/profile-picture.jpg" class="rounded-circle logo">
-                    <h1>Christiane Dragvik</h1>
-                </div>
-
+                <div class="border bg-secondary col-3 m-2">
+                    <h3>Other contacts</h3>
+                    <div class="d-flex">
+                        <img width="40px"src="images/proflie.png">
+                        <p>Username</p>                        
+                    </div>
+                    <div class="d-flex">
+                        <img width="40px"src="images/proflie.png">
+                        <p>Username</p>                        
+                    </div>
+                    <div class="d-flex">
+                        <img width="40px"src="images/proflie.png">
+                        <p>Username</p>                        
+                    </div>
+                    <div class="d-flex">
+                        <img width="40px"src="images/proflie.png">
+                        <p>Username</p>                        
+                    </div>
+                    <div class="d-flex">
+                        <img width="40px"src="images/proflie.png">
+                        <p>Username</p>                        
+                    </div>
 
-                <div class="border bg-secondary col-4">
-                    <p>Christiane's contacts</p>
-                    <img>
                 </div>
             </div>
 
+            <br>
+            <br>
             <div>
+                <h1>Share something</h1>
```

Comment:
Hello! You should only use 1, h1 tag per .html file. The h1 should be what the page is about

Reply:
Thanks for the feedback! Have made that mistake multiple times - maybe it will stick after this time 😅